registration/include/pcl/registration/gicp.h

```diff
@@ -375,15 +375,15 @@ class GeneralizedIterativeClosestPoint
 
   /** \return trace of mat1^t . mat2
    * \param mat1 matrix of dimension nxm
-   * \param mat2 matrix of dimension nxp
+   * \param mat2 matrix of dimension mxn
```

Comment:
Shouldn't mat2 also be `nxm`? If mat1 is `nxm` before being transposed, and `mxn` after?

Reply:
You are right, I've created a new commit.